examples/03_evaluate/als_movielens_diversity_metrics.ipynb

```diff
@@ -36,13 +36,16 @@
     "We evaluate the results obtained with two approaches: using the ALS recommender algorithm vs. a baseline of random recommendations. \n",
```

Comment:
<p>Something it would be nice is to talk about why diversity metrics are important for a customer point of view. Maybe also talk about the long tail items: https://github.com/microsoft/recommenders/blob/main/GLOSSARY.md</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
addressed

Reply:
<p>I am not sure that items in the tail are always highly profitable, they could be cheap because there is low demand for them. Maybe it'd be better to qualify the statement e..g say "oftentimes" or something to that effect.   </p>